src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -68,7 +68,9 @@ public class Program
 #### Metrics
 
 > **Note**
-> Metrics are not available for .NET Framework.
+> Metrics are only available for .NET Framework when traces are recorded. This requires:
+> 1. Tracing to be enabled by calling `.AddHttpClientInstrumentation()` on `TracerProviderBuilder`
+> 2. [If a sampler is in place, return `SamplingDecision.RecordAndSampled` for `ShouldSample`](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/extending-the-sdk/README.md#filtering-processor)
```

Comment:
nits:
_"If a sampler"_ is not right, as there is **always** a sampler! Suggestion:
The sampler must return `RecordAndSample` for *ever* `Activity` in order for the Metrics to be accurate. If some spans (Activity) are sampled, but others not, it'll affect Metrics accuracy, as Metric Tags (Attributes) are generated based on the Activity.

Reply:
also, the user provided Filters can also affect this, https://github.com/open-telemetry/opentelemetry-dotnet/blob/06c704b99f8c101299e0f2b90f6f11a0f15da065/src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs#L274.

1. Enable tracing.
2. Sampler returns RecordAndSample
3. No user defined filters.

Reply:
I suggest revisit the design of relying on Activity for generating metrics, and instead produce Metrics independently.
The approach done in this PR is more like a "Span2Metrics" processor, and can done as a regular `ActivityProcessor` instead, so it'll be easy to convey that Metrics are only as good as the underlying Activity.

Could you check how big of a change it'd to produce metrics independently? (It may not be as bad to rely on Activity for duration alone...but not for the whole tags.)